setup.py

```diff
@@ -20,8 +20,8 @@
 MAJOR = 5
 MINOR = 0
 MICRO = 0
-PRERELEASE = "rc2"
-IS_RELEASED = True
+PRERELEASE = ""
```

Comment:
let's continue bumping up - even if we know that the next release will be the final release.

```suggestion
PRERELEASE = "rc3"
```